Source/ThirdParty/SDL/cmake/sdlchecks.cmake

```diff
@@ -104,7 +104,7 @@ endmacro()
 macro(CheckOSS)
   if(OSS)
     # Urho3D - bug fix - should use different variables for different checks, however, we replace the whole checks with find_package() approach for consistency sake
-    find_package (OSS)
+    find_package (OSS QUIET)
     if(OSS_FOUND)
```

Comment:
These QUIET's are good, please note changes to third party sources/scripts with an ATOMIC BEGIN/ATOMIC END comment block and description of change.  Otherwise, easy to lose changes when updating from upstream.